utest/run.py

```diff
@@ -48,40 +58,19 @@ def get_tests(directory=None):
         elif testfile.match(name):
             modname = os.path.splitext(name)[0]
             if modname in imported:
-                print("Test module '%s' imported both as '%s' and '%s'. "
-                      "Rename one or fix test discovery."
-                      % (modname, imported[modname],
-                         os.path.join(directory, name)), file=sys.stderr)
+                print(
```

Comment:
this formatting was done automatically by python black

Reply:
Both old and new formatting look a bit odd for me. Perhaps using f-string here would make it better but since this isn't in actual production code I don't care too much.

Could also consider using `pathlib` instead of `os.path`, but then there would be even more changes and it would probably be better to have all these `utest/run.py` changes in a separate PR.